src/wwwroot/controllers/PlanCtrl.js

```diff
@@ -174,6 +184,42 @@
       }      
     }
 
+    function isUpgradeablePlan(limit){
```

Comment:
We are trying to use https://google.github.io/styleguide/javascriptguide.xml -> "Method and property definitions" spacing

Reply:
Perfect, I will read it.